python/gpusim_search.py

```diff
@@ -17,7 +17,9 @@ def main():
         similarity_cutoff = 0
 
         fp_binary = smiles_to_fingerprint_bin(smiles)
```

Comment:
Let's change this to:
`fp_binary, canon_smiles = smiles_to_fingerprint_bin(smiles)`
Then we can continue to use fp_binary down below (what we actually want to pass), and not the first member of a weirdly named tuple.

Reply:
Thanks, I have fixed it.